memcached.c

```diff
@@ -3396,6 +3408,7 @@ static inline void get_conn_text(const conn *c, const int af,
             protoname = IS_UDP(c->transport) ? "udp6" : "tcp6";
             break;
 
+#ifndef DISABLE_UNIX_SOCKET
```

Comment:
think I'd like to add a scope (`{ }`)  around the code under case AF_UNIX: so pathlen can be moved in there and reduce two ifdef's to one.